perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -287,6 +282,12 @@ sub confirm_subscription : Private {
 
     return unless $p->state eq 'unconfirmed' || $already_confirmed;
 
+    if (($p->category eq 'Bulky collection' || $p->category eq 'Small items collection') && $c->cobrand->suppress_report_sent_email($p)) {
```

Comment:
Is there a reason to limit this to bulky? Could conceivably see this happening with garden, and when it was in its old location it didn't have this check. A test for it being sent for staff would be good too.